geometry_msgs/Pose pose
geometry_msgs/Vector3 size
bool add
---
bool status
